css/style.css

```diff
@@ -0,0 +1,2929 @@
+      background-size: 400px 100px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 425px 150px;
+      background-size: 480px 340px ;
+  }
+
+}
+
+@media (max-width: 900px) {
+  .container {
+      max-width: 900px;
+  }
+
+  .top {
+      text-align: center;
+      background: url(../img/tesla-fon.png)  no-repeat;
+      background-position: 50% 0;
+      background-size: 900px 467px;
+  }
+
+  .charset__speed {
+      margin-left: 70px;
+  }
+
+  .button__outline {
+      margin-left: 0;
+      margin-right: 0;
+  }
+
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+      background-size: 370px 100px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 430px 150px;
+      background-size: 450px 300px ;
+  }
+
+}
+
+@media (max-width: 880px) {
+  .container {
+      max-width: 880px;
+  }
+
+  .top__wrapper-charset {
+    margin-right: 0;
+  }
+
+  .charset__speed {
+      margin-left: 100px;
+  }
+
+  .top__wrapper-item {
+      margin-right: 110px;
+  }
+
+  .button__outline {
+      padding-left: 0;
+      padding-right: 20px;
+  }
+
+  .button__red {
+      padding-left: 25px;
+      padding-right: 25px;
+      margin-right: 0;
+  }
+
+  .middle {
+      max-width: 880px;
+      margin-top: 30px;
+  }
+
+  .middle__inner {
+      align-content: space-between;
+      background: url(../img/tesla-fon-ipad-2.png) no-repeat;
+      background-size: 880px auto;
+  }
+
+  .middle__fon {
+      display: flex;
+      display: inline-block;
+  }
+
+  .middle__title {
+      margin-top: 11px;
+      margin-left: 434px;
+  }
+
+  .middle__info {
+      margin-top: 400px;
+      max-width: 880px;
+      height: 490px;
+      margin-top: 405px;
+  }
+
+  .middle__info-row {
+      display: flex;
+      flex-direction: row;
+  }
+
+  .middle__content {
+      margin-left: 0;
+  }
+
+  .middle__row {
+      margin-right: 0;
+      padding-right: 30px;
+  }
+
+  .middle__info-title {
+      margin-top: 57px;
+      margin-left: 32px;
+  }
+
+  .middle__info-text {
+      width: 320px;
+      margin-right: 0;
+      margin-top: 22px;
+      margin-left: 32px;
+  }
+
+  .middle__row {
+      margin-right: 70px;
+      margin-top: 33px;
+      margin-left: 50px;
+  }
+
+  .middle__screen {
+      padding-bottom: 0;
+      padding-top: 8px;
+  }
+
+  .middle__text {
+      padding-bottom: 25px;
+  }
+
+  .middle__btn {
+      margin-left: 152px;
+      margin-top: 17px;
+  }
+
+  .button__btn {
+      display: flex;
+      align-items: center;
+      margin-left: 150px;
+      margin-right: 0;
+      padding-left: 0;
+      margin-top: 15px;
+  }
+
+  .button__btn-img {
+      width: 32px;
+      height: 32px;
+  }
+
+  .button__btn-word {
+      width: 150px;
+      margin-left: 15px;
+      margin-right: 23px;
+      font-weight: 600;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)